screen/wallets/WalletDetails.tsx

```
@@ -146,6 +153,43 @@
+      }
+    } catch (error) {
+      console.error('Failed to set receiveBitcoinIntent:', error);
+    }
+  };
+
+  useEffect(() => {
+    const checkEnvironment = async () => {
+      try {
+        const systemVersion = await DeviceInfo.getSystemVersion();
+        let majorVersion = parseInt(systemVersion, 10);
+        if (isNaN(majorVersion)) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)